docs/source/operations/howto/sso-oidc.rst

```diff
@@ -32,12 +32,16 @@ The provider will give you a **client id** and a **client secret**. Deploy them
 Set up user creation
 ====================
 
-When a user first logs in through OIDC, they are assigned a username and a channel. 
-
-You must select a channel to which the OIDC users will be assigned with ``oidc.users.channel``.
+When a user first logs in through OIDC, they are assigned a username.
 
 The username is based on user email; the ``oidc.users.appendDomain`` flag controls whether email domain is included.
 
+Depending on the ``oidc.users.requireApproval`` flag you'll need to assign a value or not to ``oidc.users.channel``.
+If the flag is false you must select a channel to which the OIDC users will be assigned with ``oidc.users.channel``.
+If the flag is true then you must leave the ``oidc.users.channel`` empty. This will then trigger the users approval behavior.
+Users using OIDC to connect for the first time will have to wait for admin approval.
```

Comment:
I think this paragraph needs rephrasing, it's not very clear

Maybe we could express it as a this as a two-items list, eg

> There are two ways to blah blah
> - by default (`oidc.users.requireApproval=true`), bloh bloh
> - else, set `oidc.users.channel`, bluh bluh